clusterloader2/testing/density/config.yaml

```diff
@@ -1,11 +1,19 @@
+# ASSUMPTIONS:
+# - Underlying cluster should have 100+ nodes.
+# - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
+
 #Constants
 {{$NODES_PER_NAMESPACE := 100}}
-{{$PODS_PER_NODE := 30}}
+{{$PODS_PER_NAMESPACE := 3000}}
+{{$MIN_LATENCY_PODS := 500}}
+{{$MIN_SATURATION_PODS_TIMEOUT_S := 120}}
```

Comment:
I don't think that's needed.

Reply:
I'm still using it to calculate saturationRCTimeout. Value changed to 180.